data/tests/TestPhysicsObject.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import random
+from data.Kinematics import PhysicsObject
+from data.Vector import Vector
+from data.Player import Player
+from data.Pitch import Pitch
+from mock import Mock
+
+class TestPhysicsObject(unittest.TestCase):
+
+    MAX_RANDOM_VELOCITY = 25
+    MIN_RANDOM_POSITION = 0
+    MAX_RANDOM_POSITION = 100
+    MIN_RANDOM_RADIUS = 5
+    MAX_RANDOM_RADIUS = 5
+
+    def test_friction(self):
+        po = PhysicsObject(50, 50, 1, 50, [(25, 75), (25, 75)])
+        for i in range(0, 1000):
+            po._vel = Vector(random.randint(0, self.MAX_RANDOM_VELOCITY), random.randint(0, self.MAX_RANDOM_VELOCITY))
```

Comment:
It's not preferred to use random numbers because we cannot say if it is working thanks to lucky or it is good. But I can accept it.